test/integration/instance.test.js

```diff
@@ -323,6 +326,9 @@ describe(Support.getTestDialectTeaser('Instance'), () => {
         return this.User.findById(1).then(user1 => {
           return user1.decrement('aNumber', { by: 2 }).then(() => {
             expect(user1.aNumber).to.be.equal(-2);
+            return user1.decrement('bNumber', { by: 2, returning: false }).then(user3 => {
+              expect(user3.bNumber).to.be.equal(0);
+            });
```

Comment:
Add a test that it returns updated instance by default. `returnValues.returning` is enabled only for postgres so it should not fail for other dialect.

Reply:
The outer test accounts for the default case (`expect(user1.aNumber).to.be.equal(-2)`). If the code is unclear, I could break out the `returning: false` test case into a seperate block. I was mostly just trying to follow the existing test structure which is why I ended up nesting the tests. (I prefer chaining promises rather than nesting them anyways.)

Reply:
No thats ok, I totally missed that